### ISSUE TITLE
About mf channel

### ISSUE THREAD

**Issue by yaoyuan10475** [UCSB-VRL/MethaneMapper-Spectral-Absorption-aware-Hyperspectral-Transformer-for-Methane-Detection#16]
Hello dear author:
           Does the mf channel contain refined methane information? When I visualized mf, as shown in the figure below, no methane information was found. At the same time I also visualized the mask.Or am I visualizing it wrong?
![ang20160917t200444_rdn_v1n2_img_80_5](https://github.com/UCSB-VRL/MethaneMapper-Spectral-Absorption-aware-Hyperspectral-Transformer-for-Methane-Detection/assets/74589851/8addd3f3-cb73-4f30-a495-cde02d33a598)
![ang20160917t200444_rdn_v1n2_img_80_5_mask](https://github.com/UCSB-VRL/MethaneMapper-Spectral-Absorption-aware-Hyperspectral-Transformer-for-Methane-Detection/assets/74589851/75bf713e-af2e-43c8-9028-625719e50492)

**Comment by satish1901** [comment]
Can you give me the patch number or image_id or filename. I will confirm this and get back to you?

**Comment by yaoyuan10475** [comment]
Thank you for your reply. The name of my visualized mf file is ang20160917t200444_rdn_v1n2_img_80_5.npy. This is my code for visualization.
mf_img = np.load('ang20160917t200444_rdn_v1n2_img_80_5.npy')
mf_img = torch.from_numpy(mf_img)
mf_img = F.normalize(mf_img.unsqueeze(0),mean=[0.49631513], std=[0.08298772])
mf_img = np.asarray(mf_img.squeeze(0))
plt.imshow(mf_img)
plt.show()

**Comment by satish1901** [comment]
Hello Yuan,

Sincere apologies for so much delay in response. 
I think there is something wrong with the data sample. I tested on another example, here is how the results looks like:

![Sample_Visualizations](https://github.com/user-attachments/assets/f0adefa2-fd6f-4888-abc1-728674f9dbad)

flight line name: ang20150420t180624_rdn_v1f

download link: https://avng.jpl.nasa.gov/avng/y15/ang20150420t180624.tar.gz

let me know if you still see the problem
